x/poolmanager/router_test.go

```diff
@@ -1488,30 +1551,88 @@ func (s *KeeperTestSuite) TestSingleSwapExactAmountIn() {
 		poolId                 uint64
 		poolCoins              sdk.Coins
 		poolFee                sdk.Dec
+		takerFee               sdk.Dec
 		tokenIn                sdk.Coin
 		tokenOutDenom          string
 		tokenOutMinAmount      sdk.Int
 		expectedTokenOutAmount sdk.Int
-		swapWithNoTakerFee     bool
 		expectError            bool
 	}{
 		// Swap with taker fee:
 		//  - foo: 1000000000000
 		//  - bar: 1000000000000
 		//  - spreadFactor: 0.1%
-		//  - takerFee: 0.15%
+		//  - takerFee: 0.25%
```

Comment:
This change was made for two reasons:
- 0.15% was originally the default value so it was important to test something different (no longer the case)
- 0.25% is large enough to test exact rounding behavior around the default input size